src/app/cube/details/error-status/error-status.component.html

```diff
@@ -4,16 +4,16 @@
       <div class="row justify-content-center">
           <div class="col-md-12 text-center">
             <div *ngIf="statusCode">
-              <strong><h1 class="huge">{{ statusCode }}</h1></strong>
-              <div class="message">This Learning Object is currently under review.</div>
+              <strong><h1 class="huge" tabindex="0" attr.aria-label="Error status: {{ statusCode }}">{{ statusCode }}</h1></strong>
+              <div tabindex="0" class="message">This Learning Object is currently under review.</div>
               <div *ngIf="!auth.user; else unauthorized">
-                <div class="message">Try logging in to see if you have access.</div>
-                <button class="auth-button" (click)="login()">LOGIN</button>
+                <div tabindex="0" class="message">Try logging in to see if you have access.</div>
+                <button class="auth-button" (click)="login()" aria-label="Clickable Login button">LOGIN</button>
               </div>
               <ng-template #unauthorized>
-                <div class="message">You do not have access to this learning object.</div>
+                <div tabindex="0" class="message">You do not have access to this learning object.</div>
               </ng-template>
-              <a routerLink='' class="btn btn-link">Back to Home</a>
+              <a routerLink='' class="btn btn-link" aria-label="Clickable back to homepage link">Back to Home</a>
```

Comment:
Is it possible to have a link that isn't clickable? I don't know if 508 has any patterns to follow or whatnot, but it seems this could be simplified. Perhaps:

```suggestion
              <a routerLink='' class="btn btn-link" aria-label="Link back to homepage">Back to Home</a>
```
?

Reply:
Clickable should be there to add clarification for individuals using assistive technologies.